link_shorterner/src/shortener_shortener.erl

```diff
@@ -0,0 +1,63 @@
+-module(shortener_shortener).
+
+-export([init/0,
+         short/1,
+         get/1,
+         subscribe/1,
+         unsubscribe/1]).
+
+init() -> 
+    ets:new(ets_name(), [set, public, named_table]),
+    pg2:create(subs_name()).
+
+short(LongUrl) ->
```

Comment:
I'm not convinced by the name of this function. `shorten` would be more correct than `short` if it was only doing the shortening; but it's also storing and notifying subscribers.